website/templates/share_search.mako

```diff
@@ -1,6 +1,12 @@
 <%inherit file="base.mako"/>
 <%def name="title()">SHARE</%def>
 <%def name="content()">
+  <script type="text/javascript" src="/static/vendor/bower_componenets/MathJax/unpacked/MathJax.js?configi=TeX-AMS-MML_HTMLorMML"></script>
```

Comment:
This should be loaded below-the-fold. You can move this to the `javascript_bottom` def.